waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -27,22 +27,34 @@ declarePublicGauge waku_connected_peers, "Number of connected peers per directio
 logScope:
   topics = "waku node peer_manager"
 
-type
-  PeerManager* = ref object of RootObj
-    switch*: Switch
-    peerStore*: PeerStore
-    storage: PeerStorage
-
 const
   # TODO: Make configurable
   DefaultDialTimeout = chronos.seconds(10)
 
+  # Max attempts before removing the peer
+  MaxFailedAttempts = 5
+
+  # Time to wait before attempting to dial again is calculated as:
+  # initialBackoffInSec*(backoffFactor^(failedAttempts-1))
+  # 120s, 480s, 1920, 7680s
+  InitialBackoffInSec = 120
+  BackoffFactor = 4
```

Comment:
Just wondering what people's views are on making this configurable by the operator. To me, it allows for more aggressive dialing behaviour. wdyt?

Reply:
mm these are some default safe values that i've tested. what do you mean by configurable? new cli flags? not sure if that would be to low level for an operator? note though that they can be changed when creating the peermanager.

Reply:
yeah, I mean cli flags :)